File: Updog.Application/Comment/UseCases/FindByPost/CommentFinderByPostParams.cs
namespace Updog.Application {
    /// <summary>
    /// Params to find a page of comments for a post.
    /// </summary>
    public sealed class CommentFinderByPostParams {
        #region Properties
        public int PostId { get; }
        #endregion

        #region Constructor(s)
        /// <summary>
        /// Create a new set of comment finder by post params.
        /// </summary>
        /// <param name="postId">The post ID to look for.</param>
        public CommentFinderByPostParams(int postId) {
            PostId = postId;
        }
        #endregion
    }
}

File: Updog.Client/src/comment/interactors/find-by-post/comment-finder-by-post-params.ts
/**
 * Params for the comment finder by user interactor.
 */
export class CommentFinderByPostParams {
    /**
     * Create a new set of parameters for the post finder by user interactor.
     * @param postId The ID of the post to get comments for.
     * @param paginationInfo The paging info to send over to the back end.
     */
    constructor(public postId: number) {}
}
